esp/services/ws_ecl/ws_ecl_service.cpp

```diff
@@ -1533,9 +1536,13 @@ int CWsEclBinding::getGenForm(IEspContext &context, CHttpRequest* request, CHttp
     xform->setParameter("includeSoapTest", "1");
     xform->setParameter("useTextareaForStringArray", "1");
 
+#ifdef _CONTAINERIZED
```

Comment:
Could be as simple as:

```
#ifdef _CONTAINERIZED
    bool isRoxie = wsecl->connMap.getValue(wsinfo.qsetname)!=nullptr;
#else
    Owned<IConstWUClusterInfo> clusterInfo = getTargetClusterInfo(wuinfo.qsetname);
    bool isRoxie = clusterInfo && (clusterInfo->getPlatform() == RoxieCluster);
#endif

xform->setParameter("includeRoxieOptions", isRoxie ? "1" : "0");
```

Reply:
quite possibly.  should be fixed as part of HPCC-25850.

Reply:
Yes - there are separate jiras for fixing the issues.